lil_aretomo/aretomo.py

```diff
@@ -68,7 +70,7 @@ def align_tilt_series(
         gpu_ids=gpu_ids,
     )
     aretomo_output = AreTomoOutput(tilt_series_file=tilt_series_file, reconstruction_file=reconstruction_file)
-    if aretomo_output.contains_alignment_results is True:
+    if aretomo_output.contains_alignment_results is True and skip_if_completed is True:
         print(f'{basename} has already been aligned in {output_directory}, skipping...')
```

Comment:
let's not print in the function, it's a bit unexpected and might not be wanted if someone is building an application - just execute the behaviour specified in the func :)

Reply:
donee